README.md

```diff
@@ -37,6 +37,49 @@ For static use cases, wRPC provides [WIT] binding generators for:
 
 wRPC fully supports the unreleased native [WIT] `stream` and `future` data types along with all currently released WIT functionality.
 
+## Quickstart
+
```

Comment:
Could we add a small note here that there are 2 different kinds of examples - native wRPC applications, using a particular wRPC transport (currently, NATS only) and *generic* Wasm components, that run in a Wasm runtime?